src/main/java/apoc/metrics/Metrics.java

```diff
@@ -0,0 +1,111 @@
+        }
+    }
+
+    @Procedure
+    @Description("apoc.metrics.list() - get a list of available metrics")
+    public Stream<Neo4jMetric> list() {
+        Object metricsSetting = ApocConfiguration.allConfig.get("dbms.directories.metrics");
+
+        File metricsDir = new File(metricsSetting == null ?
+                ApocConfiguration.allConfig.get("unsupported.dbms.directories.neo4j_home") + File.separator + "metrics"
+                : metricsSetting.toString());
+
+        if (!metricsDir.exists() || !metricsDir.isDirectory()) {
+            return Stream.empty();
+        }
+
+        FileUtils.checkReadAllowed(metricsDir.toURI().toString());
+
+        return Arrays.asList(metricsDir.list())
+                .stream()
+                .filter(s -> s.endsWith(".csv"))
+                .map(metricFile -> {
+                    String metricName = metricFile.substring(0, metricFile.length() - 4);
+                    File f = new File(metricsDir + File.separator + metricFile);
+
+                    return new Neo4jMetric(metricName, f.lastModified(), f.getAbsolutePath());
+                });
+    }
+
+    @Procedure
+    @Description("apoc.metrics.get(metric_name) - retrieve a system metric")
+    /**
+     * This method is a specialization of apoc.load.csv, it just happens that the directory and file path
+     * are local.  For Neo4j metrics, see: https://neo4j.com/docs/operations-manual/current/monitoring/metrics/reference/
+     *
+     * The reason it's not suitable for users to use regular apoc.load.csv to get at the metrics file is that from the
+     * outside they have no way of knowing the home path where they're located, and apoc.load.csv doesn't allow
+     * relative paths.
+     */
+    public Stream<Neo4jMetricEntry> get(
+            @Name("metric_name") String metricName,
+            @Name(value = "config",defaultValue = "{}") Map<String, Object> config) {
+        // These config parameters are generally true of Neo4j metrics.
+        config.putIfAbsent("sep", ",");
+        config.putIfAbsent("header", true);
+
+        String neo4jHome = ApocConfiguration.allConfig.get("unsupported.dbms.directories.neo4j_home");
+        String url = "file://" + neo4jHome + File.separator + "metrics" + File.separator + metricName + ".csv";
+
+        FileUtils.checkReadAllowed(url);
+
+        return new LoadCsv().csv(url, config).map(entry -> {
+            long t = 0;
+            double v = 0.0;
+
+            try {
+                t = Long.parseLong(entry.map.get("t").toString());
+            } catch (NumberFormatException exc) {
+                // Load CSV will give you the header row.
+                return null;
+            }
+
+            try {
+                v = Double.parseDouble(entry.map.get("value").toString());
+            } catch(NumberFormatException exc) {
+                return null;
+            }
+
+            return new Neo4jMetricEntry(t, v, metricName);
+        }).filter(p -> p != null);
```

Comment:
Talked to Dana Canzano -- this odd little bit is necesary due to a bug in the way that neo4j does CSV metrics.  Apparently after restarts and some other situations, the CSV header will be inserted into the file a second or a third time.   Note above that if the Double.parseDouble fails, we return null, and here they get filtered out. This does come up in practice.